Mathlib/Algebra/CharP/Algebra.lean

```diff
@@ -61,6 +61,10 @@ theorem charP_of_injective_algebraMap' (R A : Type*) [Field R] [Semiring A] [Alg
     [Nontrivial A] (p : ℕ) [CharP R p] : CharP A p :=
   charP_of_injective_algebraMap (algebraMap R A).injective p
 
+theorem charP_of_injective_algebraMap'' (R : Type*) {A : Type*} [CommRing R] [CommRing A]
```

Comment:
I think you can generalize the theorem above in place, your new version is strictly more general.